redaxo/src/core/.phpstorm.meta.php

```diff
@@ -5,12 +5,102 @@
 override(
     \rex::getProperty(0),
     map([
+        'console' => \rex_console_application::class,
         'login' => \rex_backend_login::class,
-        'user' => \rex_user::class,
         'timer' => \rex_timer::class,
+        'user' => \rex_user::class,
     ])
 );
 
```

Comment:
Link zum blog post als kommenat mit rein, für bissl doku

```suggestion
// https://blog.jetbrains.com/phpstorm/2019/02/new-phpstorm-meta-php-features/
```